single_tech_samples/synapseanalytics/sample2_synapseserverless/scripts/deploy_synapse_artifacts.sh

```diff
@@ -143,6 +143,25 @@ do
     echo "$provision_state: checking again in 10 seconds..."
 done
 
+addPackageToSynapseWorkspace(){
+    echo "$(date):Synapse Workspace: Adding Wheel file as a workspace package"
+    az synapse workspace-package upload --workspace-name "${SYNAPSE_WORKSPACE_NAME}" \
+        --package ./synapseartifacts/workspace/workspace_packages/adlsaccess-1.0-py3-none-any.whl
+}
+
+attachingPackageToSparkPool(){
```

Comment:
nit (feel free to discard): I know this is a bit of an overkill comment but I would suggest changing the name of this function to **attachPackageToSparkPool** (removing the infinitive tense), for consistency with other function names on this script

Reply:
Agree. Corrected